_includes/collection-item.html

```diff
@@ -1,28 +1,33 @@
-<li class="usa-collection__item">
+<div class="usa-prose padding-right-4">
```

Comment:
does the name of this component still make sense if you're pulling the collection item class out? also if these are items in a list, they should still be `<li>`s

Reply:
This change was mostly to keep the look and feel of the existing cg-site--I'll defer to you on this--pages way or pre-existing cg-site look?  They probably shouldn't be in a list as these are multi-part paginated code blocks.  Semantically, it's fine either way.